frontend/server/src/DAO/Problems.php

```diff
@@ -1132,4 +1132,27 @@ public static function hasBeenUsedInCoursesOrContests(
         }
         return $problems;
     }
+
+    final public static function getAmountOfProblems(string $problemLevelTag): int {
+        $sql = 'SELECT
+                    count(Problems.problem_id) as pa
```

Comment:
nit:

```suggestion
                    COUNT(Problems.problem_id) AS pa
```

todas las palabras clave de SQL van en mayúsculas.